### PR TITLE
OADP-2493 created new PR without attributes include

### PR DESCRIPTION
**Pull request by shdeshpa07**
## Jira 

* [OADP-2493](https://issues.redhat.com/browse/OADP-2493)

Created this PR because the original [PR](https://github.com/openshift/openshift-docs/pull/79683) was failing during merge.
I have removed include::_attributes/common-attributes.adoc[] from the module oadp-about-disable-node-agent-dpa.adoc which was causing the merge to fail.
This PR is approved by QE and has gone through OCP peer review.  Resubmitting the PR again for merging. 

##  Version

* OCP 4.13 → OCP 4.17

## Preview

[AWS - About disabling node agent in the DataProtectionApplication](https://80039--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-about-disable-node-agent-dpa_installing-oadp-aws)

[Azure - About disabling node agent in the DataProtectionApplication](https://80039--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure#oadp-about-disable-node-agent-dpa_installing-oadp-azure)

[GCP - About disabling node agent in the DataProtectionApplication](https://80039--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp#oadp-about-disable-node-agent-dpa_installing-oadp-gcp)

[MCG - About disabling node agent in the DataProtectionApplication](https://80039--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg#oadp-about-disable-node-agent-dpa_installing-oadp-mcg)

[ODF - About disabling node agent in the DataProtectionApplication](https://80039--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs#oadp-about-disable-node-agent-dpa_installing-oadp-ocs)

## QE Review

* [x] QE has approved this change.

### CODE REVIEW

**Comment by shdeshpa07** [comment]
Hi @anarnold97 - I have created this new PR because the [original PR](https://github.com/openshift/openshift-docs/pull/79683)  was failing during merge. 

Kerry pointed this to me during the merge review and asked me to resubmit the PR. I have now removed include::_attributes/common-attributes.adoc[] from the module oadp-about-disable-node-agent-dpa.adoc which was causing the merge to fail.

QE and OCP review is done for this PR. Would you please give this PR  a quick review? If all looks okay, I will add the merge label to the PR.

cc: @kcarmichael08 

Thanks.

**Comment by shdeshpa07** [comment]
This is the original PR and the commit.

0fbcd91a67502c9ded94106d63acc9d7d27ab27e  xref: https://github.com/openshift/openshift-docs/pull/79683

**Comment by shdeshpa07** [comment]
@kcarmichael08  -  Sending this PR for OCP peer review again. Thanks.

File: modules/oadp-about-disable-node-agent-dpa.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-about-disable-node-agent-dpa_{context}"]
+= About disabling node agent in the DataProtectionApplication
```
Comment (xenolinux) [inline modules/oadp-about-disable-node-agent-dpa.adoc:11]:
```suggestion
= Disabling the node agent in DataProtectionApplication
```

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc [hunk 2]
```diff
@@ -37,6 +37,8 @@ include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
 include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```
Comment (xenolinux) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc:41]:
```suggestion

```
This link belongs in the **additional resources** section. And then to point users to this section [(guidelines),](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) you can say:
>"For more information, see "Running tasks in pods using jobs"."

In the `oadp-about-disable-node-agent-dpa.adoc` module

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc [hunk 3]
```diff
@@ -34,6 +34,8 @@ include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
 include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```
Comment (xenolinux) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc:38]: Same comment: This link belongs in the additional resources section.

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc [hunk 4]
```diff
@@ -35,6 +35,8 @@ include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-gcp-wif-cloud-authentication.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```
Comment (xenolinux) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc:39]: Same comment: This link belongs in the additional resources section.

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc [hunk 5]
```diff
@@ -41,6 +41,8 @@ include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
 include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```
Comment (xenolinux) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc:45]: Same comment: This link belongs in the additional resources section.

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc [hunk 6]
```diff
@@ -45,6 +45,8 @@ include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-creating-object-bucket-claim.adoc[leveloffset=+2]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```
Comment (xenolinux) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc:49]: Same comment: This link belongs in the additional resources section.

**Review by xenolinux — COMMENTED** [review]
Before requesting for the merge, I request to consider the following comments and suggestions. Please let me know if you have any questions. Thanks!

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc (hunk 3 above)
Comment (shdeshpa07) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc]: I have included this change in the PR.

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc (hunk 5 above)
Comment (shdeshpa07) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc]: I have included this change in the PR.

**Comment by shdeshpa07** [comment]
> Before requesting for the merge, I request to consider the following comments and suggestions. Please let me know if you have any questions. Thanks!
> 
> /remove-label peer-review-needed /remove-label peer-review-in-progress /label peer-review-done

Thank you for the review. I have incorporated all of the changes your suggested. Sending it for merge review. Thanks.

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc (hunk 4 above)
Comment (shdeshpa07) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc]: applied the change

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc (hunk 6 above)
Comment (shdeshpa07) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc]: applied the change

**Review by sheriff-rh — APPROVED** [review]
This is great work - thank you for providing all of the preview links too! 

Merging to 4.13+.

**Pull request merged**